toonz/sources/toonzfarm/tfarmcontroller/tfarmcontroller.cpp

```diff
@@ -90,14 +97,16 @@ TFilePath getLocalRoot() {
   TFilePath lroot;
 
 #ifdef _WIN32
-  TFilePath name(L"SOFTWARE\\OpenToonz\\OpenToonz\\1.0\\TOONZROOT");
+  QString regpath = QString::fromStdString("SOFTWARE\\OpenToonz\\OpenToonz\\" + std::string(applicationVersion) + "\\FARMROOT");
```

Comment:
As far as I can tell, TFilePath already has a constructor that takes an STL std::string, so I don't think there is a need to convert to a QString.

This should be all you need:

```
const std::string regpath = "SOFTWARE\\OpenToonz\\OpenToonz\\" + std::string(applicationVersion) + "\\FARMROOT";
const TFilePath name = regpath;
```

Also, it might be useful to replace the hard-coded "OpenToonz" in the strings with the _applicationName_ variable, since having it declared above is kind of misleading if changing it would have no effect. In that case you could do this:

```
const std::string regpath = "SOFTWARE\\" + std::string(applicationName) + "\\" + std::string(applicationName) + "\\" + std::string(applicationVersion) + "\\FARMROOT";
const TFilePath name = regpath;
```

If you are inclined to learn about std::stringstream or format strings, you could go into that if you prefer, but simple concatenation like in my example should work just fine in our case.